install/kubernetes/helm/istio-remote/templates/endpoints.yaml

```diff
@@ -100,6 +100,27 @@ subsets:
     port: 443
     protocol: TCP
 ---
+{{- if .Values.global.ingressGatewayEndpoint }}
```

Comment:
Are we going to ship with both ingress and gateway ? Maybe get rid of istio-ingress ?

Reply:
@costinm my understanding is we are shipping only v3 apis (i.e. ingress won't be present).

Reply:
This was there to get the pilot ingress tests to run--you can see it's only included if the helm input includes the ingress-gateway endpoint IP.  

The selectorless service and endpoints are removed with the merge of the auto-sidecar injection chart.  The multicluster e2e test setup now need to be reworked a lot for deployment of apps on remote.  IMO, the selectorless services & endpoints could coexist with the auto-sidecar diffs (maybe as subcharts in istio-remote).  I'll see if it's not too big of a change in the current e2e to get it to work without services/endpoints in istio-remote.